pkg/sources/legacy_reporters.go

```
@@ -12,13 +12,12 @@ type ChanReporter struct {
 	Ch chan<- *Chunk
 }
 
-func (c ChanReporter) ChunkOk(ctx context.Context, chunk Chunk) error {
-	return common.CancellableWrite(ctx, c.Ch, &chunk)
+func (c ChanReporter) ChunkOk(ctx context.Context, chunk Chunk) {
+	_ = common.CancellableWrite(ctx, c.Ch, &chunk)
```

Can you leave a comment explaining what kind of errors can get swallowed here and why that's ok to do?